bodhi/tests/server/services/test_graphql.py

```diff
@@ -92,3 +92,63 @@ def test_enumfields(self):
                 }]
             }
         }
+
+    def test_specificReleases(self):
+        """Testing specificReleases."""
+        client = Client(schema)
+        release = Release(
+            name='F22', long_name='Fedora 22',
+            id_prefix='FEDORA', version='22',
+            dist_tag='f22', stable_tag='f22-updates',
+            testing_tag='f22-updates-testing',
+            candidate_tag='f22-updates-candidate',
+            pending_signing_tag='f22-updates-testing-signing',
+            pending_testing_tag='f22-updates-testing-pending',
+            pending_stable_tag='f22-updates-pending',
+            override_tag='f22-override',
+            branch='f22',
+            package_manager=PackageManager.dnf,
+            testing_repository='updates-testing')
+
+        self.db.add(release)
+        self.db.commit()
+
+        executed = client.execute("""{  specificReleases(idPrefix: "FEDORA"){  name  }}""")
+        assert executed == {
+            "data": {
+                "specificReleases": [{
+                    "name": "F17"
+                }, {
+                    "name": "F22"
+                }]
+            }
+        }
+
+        executed = client.execute("""{  specificReleases(name: "F17"){  id  }}""")
+        assert executed == {
+            "data": {
+                "specificReleases": [{
+                    "id": "UmVsZWFzZTox"
+                }]
+            }
+        }
+
+        executed = client.execute("""{  specificReleases(composedByBodhi: true){  name  }}""")
+        assert executed == {
+            "data": {
+                "specificReleases": [{
+                    "name": "F17"
+                }, {
+                    "name": "F22"
+                }]
+            }
+        }
+
+        executed = client.execute("""{  specificReleases(state: "current"){  name  }}""")
```

Comment:
I would also add a test case where we have multiple attributes something like 
```suggestion
        executed = client.execute("""{  specificReleases(state: "pending", composedByBodhi: false){  name  }}""")
```